components/project-card/project-card.jsx

```diff
@@ -67,10 +89,20 @@ export default React.createClass({
     }
   },
   bookmarkProject(bookmarks) {
+    ReactGA.event({
+      category: `${GA_EVENT_PREFIX}Bookmark Button`,
+      action: `Bookmarked`,
+      label: `${this.props.id} - ${this.props.title}`
```

Comment:
This label gets repeated a lot, can you extract it out as a variable at the top?